tasks/create-cert-webroot.yml

```
@@ -0,0 +1,30 @@
+---
+- name: Check if certificate already exists.
+  stat:
+    path: /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem
+  register: letsencrypt_cert
+
+- name: Create webroot directory if it doesn't exist yet
+  file:
+    path: "{{ cert_item.webroot | default(certbot_webroot) }}"
+    state: directory
+    group: www-data
+    mode: 0750
+
+- name: Generate new certificate if one doesn't exist, one host at a time
+  command: "{{ certbot_create_command }}"
+  register: generate_certificate
+  throttle: 1
+
+- name: Print reload message
+  debug:
+    msg:
+      - "****"
+      - "**** !!! MANUAL RELOAD NEEDED !!!"
+      - "**** Configuration files changed, you have to..."
+      - "**** 1. update your webserver configuration to use:"
+      - "**** 1.1. certificate in /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem"
+      - "**** 1.2. private key in /etc/letsencrypt/live/{{ cert_item.domains | first }}/privkey.pem"
+      - "**** 2. and reload it"
+      - "****"
+  when: generate_certificate.changed
```

```suggestion
  when: _certbot_generate_certificate.changed
```

Private variables should be prefixed with `_certbot_`